api/integrations/launch_darkly/services.py

```diff
@@ -164,31 +171,49 @@ def _create_mv_feature_states(
     environments_by_ld_environment_key: dict[str, Environment],
 ) -> None:
     variations = ld_flag["variations"]
+    variation_values_by_idx: dict[str, str] = {}
     mv_feature_options_by_variation: dict[str, MultivariateFeatureOption] = {}
 
     for idx, variation in enumerate(variations):
+        variation_idx = str(idx)
+        variation_value = variation["value"]
+        variation_values_by_idx[variation_idx] = variation_value
         (
             mv_feature_options_by_variation[str(idx)],
             _,
         ) = MultivariateFeatureOption.objects.update_or_create(
             feature=feature,
-            string_value=variation["value"],
+            string_value=variation_value,
             defaults={"default_percentage_allocation": 0, "type": STRING},
         )
 
     for ld_environment_key, environment in environments_by_ld_environment_key.items():
         ld_flag_config = ld_flag["environments"][ld_environment_key]
+
+        is_flag_on = ld_flag_config["on"]
+
         feature_state, _ = FeatureState.objects.update_or_create(
             feature=feature,
             environment=environment,
-            defaults={"enabled": ld_flag_config["on"]},
+            defaults={"enabled": is_flag_on},
         )
 
         cumulative_rollout = rollout_baseline = 0
 
         if ld_flag_config_summary := ld_flag_config.get("_summary"):
             enabled_variations = ld_flag_config_summary.get("variations") or {}
             for variation_idx, variation_config in enabled_variations.items():
+                if variation_config.get("isOff"):
+                    # Set LD's off value as the control value.
+                    # We expect only one off variation.
+                    FeatureStateValue.objects.update_or_create(
+                        feature_state=feature_state,
+                        defaults={
+                            "type": STRING,
+                            "string_value": variation_values_by_idx[variation_idx],
+                        },
+                    )
+
```

Comment:
Shouldn't we `continue` after this? We don't need to create the MultivariateFeatureOption, right? Because, if I understand correctly, the control will receive the correct percentage by nature of it being the amount left over from the other variations?

Reply:
Sounds good, although we can't *not* create the `MultivariateFeatureOption`, so we'll also have this value as a 0% weight variation.

Reply:
This is done.

Reply:
Hmm.. based on the discussion on slack, I think actually it is better to create the MultivariateFeatureStateValue then. I think the FE may have issues if there are more Options than States. We can just create the Value with a 0 percentage as we were previously. Apologies for the confusion.

Reply:
Reverted.